source/_integrations/amcrest.markdown

```diff
@@ -205,6 +206,132 @@ Service data attribute | Optional | Description
+  Example showing an Amcrest IP2M-841 PT camera with controls for Pan and Tilt.
+</p>
+
+
+Using the following picture-elements card code you can display a live video feed from an Amcrest camera with controls for moving the camera. Note that there can be several seconds of lag before video feed reflects camera movement.   
+
+```yaml
+type: picture-elements
+entity: camera.house_front
+camera_image: camera.house_front
+camera_view: live   # or auto for snapshot view
+elements:
+  - type: icon
+    icon: 'mdi:arrow-up'
+    style:
+      background: 'rgba(255, 255, 255, 0.25)'
+      right: 25px
+      bottom: 50px
+    tap_action:
+      action: call-service
+      service: amcrest.ptz_control
+      service_data:
+        entity_id: camera.house_front
+        movement: up
+  - type: icon
+    icon: 'mdi:arrow-down'
+    style:
+      background: 'rgba(255, 255, 255, 0.25)'
+      right: 25px
+      bottom: 0px
+    tap_action:
+      action: call-service
+      service: amcrest.ptz_control
+      service_data:
+        entity_id: camera.house_front
+        movement: down
+  - type: icon
+    icon: 'mdi:arrow-left'
+    style:
+      background: 'rgba(255, 255, 255, 0.25)'
+      right: 50px
+      bottom: 25px
+    tap_action:
+      action: call-service
+      service: amcrest.ptz_control
+      service_data:
+        entity_id: camera.house_front
+        movement: left
+  - type: icon
+    icon: 'mdi:arrow-right'
+    style:
+      background: 'rgba(255, 255, 255, 0.25)'
+      right: 0px
+      bottom: 25px
+    tap_action:
+      action: call-service
+      service: amcrest.ptz_control
+      service_data:
+        entity_id: camera.house_front
+        movement: right
+  - type: icon
+    icon: 'mdi:arrow-top-left'
+    style:
+      background: 'rgba(255, 255, 255, 0.25)'
+      right: 50px
+      bottom: 50px
+    tap_action:
+      action: call-service
+      service: amcrest.ptz_control
+      service_data:
+        entity_id: camera.house_front
+        movement: left_up
+  - type: icon
+    icon: 'mdi:arrow-top-right'
+    style:
+      background: 'rgba(255, 255, 255, 0.25)'
+      right: 0px
+      bottom: 50px
+    tap_action:
+      action: call-service
+      service: amcrest.ptz_control
+      service_data:
+        entity_id: camera.house_front
+        movement: right_up
+  - type: icon
+    icon: 'mdi:arrow-bottom-left'
+    style:
+      background: 'rgba(255, 255, 255, 0.25)'
+      right: 50px
+      bottom: 0px
+    tap_action:
+      action: call-service
+      service: amcrest.ptz_control
+      service_data:
+        entity_id: camera.house_front
+        movement: left_down
+  - type: icon
+    icon: 'mdi:arrow-bottom-right'
+    style:
+      background: 'rgba(255, 255, 255, 0.25)'
+      right: 0px
+      bottom: 0px
+    tap_action:
+      action: call-service
+      service: amcrest.ptz_control
+      service_data:
+        entity_id: camera.house_front
+        movement: right_down
+```
+
+
```

Comment:
```suggestion

```

Reply:
Happy with the formatting updates @frenck - thanks!     Please ensure this is merged coincident with the actual amcrest component merge by @pnbruckner .

Reply:
Please adjust this PR, if a PR collides later, we can always rebase and resolve merge conflicts.

Reply:
I think @GaryOkie meant this doc PR shouldn't be merged until the corresponding code PR is merged. Gary, don't worry, that's SOP.